cmake/external_mlsl.cmake

```diff
@@ -43,12 +43,17 @@ ExternalProject_Add(
     EXCLUDE_FROM_ALL TRUE
     )
 
+add_library(libmlsl INTERFACE)
 ExternalProject_Get_Property(MLSL SOURCE_DIR)
 ExternalProject_Get_Property(MLSL INSTALL_DIR)
-add_library(libmlsl INTERFACE)
 target_include_directories(libmlsl SYSTEM INTERFACE ${SOURCE_DIR}/include)
-target_link_libraries(libmlsl INTERFACE ${INSTALL_DIR}/intel64/lib/thread/libmlsl.so)
-link_directories(${INSTALL_DIR}/intel64/lib/thread)
+
+set(MLSL_LINK_LIBRARIES
+    ${INSTALL_DIR}/intel64/lib/thread/libmlsl.so
```

Comment:
in the file name replace `lib` with `${CMAKE_SHARED_LIBRARY_PREFIX}` and `.so` with `{CMAKE_SHARED_LIBRARY_SUFFIX}`